cmd/go-tpc/main.go

```diff
@@ -251,3 +275,42 @@ func main() {
 
 	cancel()
 }
+
+// registerMysqlTLSConfig constructs a `*tls.Config` from the CA, certification and key
+// paths, and register to mysql client.
+func registerMysqlTLSConfig() {
+	// Load the client certificates from disk
+	var certificates []tls.Certificate
+	if len(sslCert) != 0 && len(sslKey) != 0 {
+		cert, err := tls.LoadX509KeyPair(sslCert, sslKey)
+		if err != nil {
+			panic(fmt.Errorf("could not load client key pair, err %v", err))
+		}
+		certificates = []tls.Certificate{cert}
+	}
+
+	// Create a certificate pool from CA
+	certPool := x509.NewCertPool()
+	ca, err := os.ReadFile(sslCA)
+	if err != nil {
+		panic(fmt.Errorf("could not read ca certificate, err %v", err))
+	}
+
+	// Append the certificates from the CA
+	if !certPool.AppendCertsFromPEM(ca) {
+		panic(fmt.Errorf("failed to append ca certs"))
+	}
+
+	tlsConfig := &tls.Config{
+		MinVersion:   tls.VersionTLS10,
```

Comment:
```suggestion
		MinVersion:   tls.VersionTLS12,
```

https://www.rfc-editor.org/rfc/rfc8996.html#name-do-not-use-tls-11